sinks_output_test.go

```diff
@@ -24,88 +24,114 @@ func TestOutputSink_Check(t *testing.T) {
 
 	logs := `
 foo debug
+foo info
 foo error
 foo.bar debug
+foo.bar info
 foo.bar error
 foo.bar.baz debug
+foo.bar.baz info
 foo.bar.baz error
 foo.bar.baz1 debug
+foo.bar.baz1 info
 foo.bar.baz1 error
 `
 
 	cases := []struct {
 		Name string
 		Env  map[string]string
-		Want string
+		Want autogold.Value
 	}{{
 		Name: "error",
 		Env: map[string]string{
 			EnvLogLevel: "error",
 		},
-		Want: `
+		Want: autogold.Expect(`
 foo error
 foo.bar error
 foo.bar.baz error
 foo.bar.baz1 error
-`,
+`),
 	}, {
 		Name: "debug",
 		Env: map[string]string{
 			EnvLogLevel: "debug",
 		},
-		Want: logs,
+		Want: autogold.Expect(logs),
 	}, {
 		Name: "none",
 		Env: map[string]string{
 			EnvLogLevel: "none",
 		},
-		Want: "",
+		Want: autogold.Expect("\n"),
 	}, {
 		Name: "scope",
 		Env: map[string]string{
 			EnvLogLevel:      "error",
 			EnvLogScopeLevel: "foo.bar=debug",
 		},
-		// Should be everything except foo debug
-		Want: `
+		// Should be everything except foo debug and info
+		Want: autogold.Expect(`
 foo error
 foo.bar debug
+foo.bar info
 foo.bar error
 foo.bar.baz debug
+foo.bar.baz info
 foo.bar.baz error
 foo.bar.baz1 debug
+foo.bar.baz1 info
 foo.bar.baz1 error
-`,
+`),
```

Comment:
just confirming, so these updates are needed not because you changed behaviour, but because you introduced info logs in the input.

Reply:
Correct. The way the tests were structured, there was only error and debug and I wanted something in the middle for my new test.